rero_invenio_base/cli/es/index.py

```diff
@@ -162,8 +165,130 @@ def update_mapping(aliases, settings):
                     f'error: {excep}', fg='red')
             if res.get('acknowledged'):
                 click.secho(
-                    f'index: {index} has been sucessfully updated',
+                    f'index: {index} has been successfully updated',
                     fg='green')
             else:
                 click.secho(
                     f'error: {res}', fg='red')
+
+
+@index.command('move')
+@with_appcontext
+@es_version_check
+@click.argument('resource')
+@click.argument('old')
+@click.argument('new')
+@click.option('-t', '--templates/--no-templates', 'templates', is_flag=True,
+              default=True)
+@click.option('-v', '--verbose/--no-verbose', 'verbose',
+              is_flag=True, default=False)
+@click.option('-n', '--interval', default=1, type=int,
+              help='seconds to wait between updates')
+def move_index(resource, old, new, templates, verbose, interval):
```

Comment:
I would propose an option to not wait for completion.

Reply:
Interval == 0 no wait.